cvmfs/mountpoint.cc

```diff
@@ -212,8 +212,14 @@ bool FileSystem::CreateCache() {
     if (options_mgr_->GetValue("CVMFS_CACHE_RAM_MALLOC", &optarg)) {
       if (optarg == "arena") {
         alloc = MemoryKvStore::kMallocArena;
+      } else if (optarg == "heap") {
+        alloc = MemoryKvStore::kMallocHeap;
       }
     }
+    if (cache_bytes % 8 != 0) {
+      // Ensure 8-byte alignment for heap allocator
+      cache_bytes += 8 - (cache_bytes % 8);
```

Comment:
We have `RoundUp8` functions for 32 bit and 64 bit arguments in malloc_arena.h and malloc_heap.h.  I'd suggest to move both of these function in smalloc.h and use them from there.